celest/schedule/removal_operators.py

```diff
@@ -0,0 +1,68 @@
+
+
+import random
+
+
+def random_removal(request_handler, number_to_remove):
+
+    if number_to_remove > request_handler.number_of_scheduled_requests:
+        request_handler.unschedule_all_requests()
+        return request_handler
+
+    while (number_to_remove > 0):
+        request_index = random.randint(0, len(request_handler.requests) - 1)
```

Comment:
Is there a better implementation that ensures the same index is not checked multiple times when a request is not unscheduled?